steelthread-0/walkthrough.md

```diff
@@ -15,123 +18,295 @@ The installation has already been performed on an OpenShift cluster as per the i
+cd /tmp/custom_nodejs_stack
+cp ~/Downloads/ca.crt ./
 ```
 
-Add in the values from the secret from the Enmasse binding secret
+Create a new file called `Dockerfile` with the below contents
+
+```
+FROM eclipse/node
+ADD ./ca.crt /usr/local/share/ca-certificates/ca.crt
+RUN sudo chmod 664 /usr/local/share/ca-certificates/ca.crt
+RUN sudo update-ca-certificates
+RUN cd ${HOME} && echo yes | keytool -keystore minishift.jks -importcert -alias HOSTDOMAIN -file /usr/local/share/ca-certificates/ca.crt -storepass minishift
+```
 
-Set the following key value pairs
+Build the container image
 
 ```
-MESSAGING_SERVICE_HOST = <value from messagingHost>
-MESSAGING_SERVICE_PASSWORD = <value from password>
-MESSAGING_SERVICE_USER= <value from username>
+docker build -t che/node .
 ```
 
-Restart the booster services
+Next, you'll need to push the image to a public registry.
+For this example, a personal account on Docker Hub is used.
+Tag the image with the correct url for your personal Docker Hub account & push it.
 
 ```
-oc rollout latest nodejs-messaging-work-queue-frontend
-oc rollout latest nodejs-messaging-work-queue-worker
+docker tag che/node docker.io/<dockerhubusername>/che-node
+docker push docker.io/<dockerhubusername>/che-node
 ```
 
-# Setup HTTP connection to the messaging booster in Fuse Ignite
-- Open the Fuse ignite console and open the connections again. 
-- Click create connection
-- Find the HTTP connection
-- Set the base URL as the url exposed from the messaging front end application
-- Click validate to ensure Fuse Ignite can reach the endpoint
-- Click next 
-- Give the connection a name (booster)
-- Click create
+**IMPORTANT:** You will need to make the `che-node` image repository **public** in your Docker Hub account. You can do this from the Docker Hub web console.
 
-# Creating the integration
+![Docker Hub Public Repository](./docker_hub_public_repository.png)
 
-- Open the Fuse Ignite console
-- Click on integrations
-- Click create integration
-- Click on the Enmaase connection
-- Click on the ``` Subscribe for messages ``` option
-- Under Destination Name add ``` work-queue/worker-updates ```
-- Choose Queue as the Destination Type
-- Click Next
-- Set the Specify Output Data Type as ``` Type specification not required ``` this is the default by clicking done
-- On the ``` Choose a Finish Connection ``` screen select the messaging booster connection you created
-- Choose ```Invoke URL ``` 
-- In the URL Path set the path to ```/api/data```
-- In the method select ```GET``` and click next
-- Again under Specify Input Data Type leave it as ``` Type specification not required ``` and click done
-- Click publish at the top right
-- Choose the integration name (steel thread)
+Back in the Che Dashboard, go to the Stacks menu & clone the Node stack
+
+![Che Clone Stack](./che_stack_clone.png)
+
+Modify the stack to set the Machine image to the public image repository name i.e. `<dockerhubusername>/che-node`
```

Comment:
`<dockerhubusername>/che-node` is valid IF the image repository is docker.io as Openshift will default to this. However, if one was to use quay.io they'll need to specify the image repo e.g. `quay.io/pmccarthy/che-node`. Might be worth changing the example to `docker.io/<dockerhubusername>/che-node` as this should work regardless.

Reply:
will update to include docker.io